Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraHelper/CameraHelperCode.bind

```diff
@@ -0,0 +1,26 @@
+﻿// Creates a Camera Helper and subscribes to frames from an available frame source.
+// Make sure you have the capability webcam enabled for your app to access the device's camera.
+
+using Microsoft.Toolkit.Uwp.Helpers.CameraHelper;
+
+CameraHelper cameraHelper = new CameraHelper();
+var result = await _cameraHelper.InitializeAndStartCaptureAsync();
+
+// Camera Initialization and Capture failed for some reason
+if(result != CameraHelperResult.Success)
+{
+// get error information
+var errorMessage = result.ToString();
```

Comment:
Missing intentation :)

Reply:
Just an example showing user how to get error info. Upto them to use the info as appropriate.

Reply:
@skommireddi I think he mean indentation... ;)

Reply:
ah ok, fixed it :)